File: src/gameState/BattleState.java
package gameState;

import battle.BattleManager;
import org.newdawn.slick.GameContainer;
import org.newdawn.slick.Graphics;

public class BattleState extends AbstractGameState {
	
	@Override
	public void render(GameContainer gc, Graphics g) {
		BattleManager.render(gc, g);
		guiman.render(gc, g);
	}
	
	@Override
	public void update(GameContainer gc) {
		BattleManager.update(gc);
		guiman.update(gc);
	}
	
}

File: src/gamestate/BattleState.kt
package gamestate

import battle.BattleServer
import org.newdawn.slick.GameContainer
import org.newdawn.slick.Graphics

object BattleState : AbstractGameState() {
    //TODO("Should the BattleState be the server?")

    override fun render(gc: GameContainer, g: Graphics) {
        BattleServer.render(gc, g)
    }

    override fun update(gc: GameContainer) {
        BattleServer.update(gc)
    }

}
